contracts/TokenHub.sol

```diff
@@ -4,82 +4,23 @@ import "./interface/0.6.x/IBEP20.sol";
 import "./interface/0.6.x/ITokenHub.sol";
 import "./interface/0.6.x/IParamSubscriber.sol";
 import "./interface/0.6.x/IApplication.sol";
-import "./interface/0.6.x/ICrossChain.sol";
 import "./interface/0.6.x/ISystemReward.sol";
 import "./lib/0.6.x/SafeMath.sol";
-import "./lib/0.6.x/RLPEncode.sol";
-import "./lib/0.6.x/RLPDecode.sol";
-import "./lib/0.6.x/BytesToTypes.sol";
-import "./lib/0.6.x/Memory.sol";
 import "./System.sol";
 
```

Comment:
`claimMigrationFund` function can be deleted now

Reply:
`claimRelayerReward` function can be deleted

Reply:
removed

Reply:
removed